pandas/core/generic.py

```diff
@@ -3636,6 +3636,7 @@ def to_csv(
         decimal: str = ...,
         errors: OpenFileErrors = ...,
         storage_options: StorageOptions = ...,
+        comment: str | None = ...,
```

Comment:
Why is this string and not bool?

Reply:
It was intended to mirror the `read_csv` parameter for consistency between the two. Setting it will write the comments with that prefix.